modules/manual-configuration-image-registry-ocp.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ocp_{context}"]
+= Manually configuring {ocp} registry
+
+You can integrate {product-title} with {ocp} built-in container image registry.
+
+.Prerequisites
```

Comment:
The convention is to follow the `.Prerequisites` and `.Procedure` block titles with a blank line.  Up to you whether to revise or work toward consistency at a later date.

Ref the [procedure module template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) from the mod-docs repo.

Reply:
To keep this PR clean, I'll take that as a separate PR. Thanks.